recipes/pyramid_mako/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pyramid_mako" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0066c863441f1c3ddea60cee1ccc50d00a91a317a8052ca44131da1a12a840e2
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+    - setuptools
+  run:
+    - python >=2.7
+    - mako >=1.1.0
+    - pyramid
+
+test:
+  imports:
+    - pyramid_mako
+  # requires:
+  #   - webtest >=1.3.1
+
+about:
+  home: https://trypyramid.com
+  license: BSD-derived
```

Comment:
From the bot: 

>  * License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.
> 
> Documentation on acceptable licenses can be found [here]( https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions ).

Reply:
This is a question I'm unclear about with conda-forge policy:

What do we do if the license of the software has no SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

In this case, it's the "Pylons Licence" which, IIUC, is pretty much BSD, but not exactly.

So how DO we put that in a conda-forge recipe?

Reply:
It's always possible to have a LicenseRef as the remainder of the string is then arbitrary e.g. `LicenseRef-Pylons`